packages/devextreme/js/ui/rating.d.ts

```diff
@@ -0,0 +1,186 @@
+import {
+    EventInfo,
+    NativeEventInfo,
+    InitializedEventInfo,
+    ChangedOptionInfo,
+} from '../events/index';
+
+import Editor, {
+    EditorOptions,
+    ValueChangedInfo,
+} from './editor/editor';
+
+/** @public */
+export type RatingSelectionMode = 'continuous' | 'single';
+
+/**
+ * @docid _ui_rating_ContentReadyEvent
+ * @public
+ * @type object
+ * @inherits EventInfo
+ */
+export type ContentReadyEvent = EventInfo<dxRating>;
```

Comment:
I finally think we should not add this property.
I don't see what it's useful for in Rating